src/client/clientenvironment.cpp

```diff
@@ -287,7 +287,7 @@ void ClientEnvironment::step(float dtime)
 		if (speed > tolerance && !player_immortal) {
 			f32 damage_f = (speed - tolerance) / BS * post_factor;
 			u16 damage = (u16)MYMIN(damage_f + 0.5, U16_MAX);
-			if (damage != 0) {
+			if (!g_settings->getBool("prevent_natural_damage") != 0) {
```

Comment:
what is the != 0 for?

Reply:
the check for damage != 0 should stay, its a minetest functionality.

Reply:
yeah stupid, stupid cora :/